src/define/create-advert.ts

```diff
@@ -21,14 +21,7 @@ const createAdvert = (
 		log('commercial', errMsg);
 
 		if (!navigator.userAgent.includes('DuckDuckGo')) {
-			reportError(
-				new Error(errMsg),
-				{
-					feature: 'commercial',
-				},
-				false,
-				1 / 100,
```

Comment:
Note: previously this error was being shown to 1% of users who were served the Sentry client